config/routes.rb

```diff
@@ -1,5 +1,5 @@
 if ShopifyApp.configuration.routes_enabled?
-  ShopifyApp::Engine.routes.draw do
+  Rails.application.routes.draw do
```

Comment:
yeah the default is to have the routes which will forward to the basic controller that is part of the app. Any user of the gem will need an initializer and they can set routes false there. 

Most users will probably run the full generator which creates the controllers and copies the routes then sets the routes config to false.

I am going to do a 6.1.0 release after this so no one upgrade accidentally. (aside there is still small risk here because if the routes will be overriden by any routes in the host app. All that can happen is that a request that should have 404ed 500s)